### ISSUE TITLE
mysql cli issues, when creating databse

### ISSUE THREAD

**Issue by mjhoy** [danchoi/vnews#1]
Hello! I saw your message on the vim ML and, intrigued, followed the installation instructions on your site.

I have a potentially weird and very-local-to-my-machine issue that occurs with `vnews --create-db`.

On line 15 of
https://github.com/danchoi/vnews/blob/da8dea963b9a4d625e71dcf6c12c2a6099cfdc83/lib/vnews/sql.rb

you construct the mysqladmin command as, e.g.,:

```
mysqladmin -h localhost -u root -p foo create vnews
```

If I run this, I'm (oddly) asked for my password, to which — if entered — it then outputs `mysqladmin: Unknown command: 'foo'`. So, it's interpreting `foo` (my password) as the first command to run.

I can rewrite it like so:

```
mysqladmin create vnews -h localhost -u root -p foo
```

and it _still_ asks for my password, but this time (at least) it runs the command correctly; or, this also works:

```
mysqladmin -h localhost -u root -p create vnews
```

(Again, I am asked for the password, but it runs the command correctly.) So the issue seems to be the password is totally ignored when it's part of the arguments. Any idea why this would be?

**Comment by blakesweeney** [comment]
I don't think it's just you. I get a similar issue.

**Comment by danchoi** [comment]
Sorry this reply comes late, but I pushed a possible fix in version 0.4.4.

**Comment by mjhoy** [comment]
Yes, it seems to fix it for me.
